app/constants/urls.js

```diff
@@ -35,7 +35,7 @@ const APP_URLS = {
     PROFILE: `${SCHEME}staging-my.${DOMAIN}`,
     TASKS: `${SCHEME}staging-my.${DOMAIN}/tasks`,
     IDENTITY: `${SCHEME}staging-my.${DOMAIN}/identity`,
-    API_BACKEND: `${SCHEME}staging-api.${DOMAIN}`,
+    API_BACKEND: `http://localhost:3000`,
```

Comment:
revert changes

Reply:
This is development config that's why added this